framework/kibocommerce/api/utils/cookie-handler.ts

```diff
@@ -0,0 +1,52 @@
+import { KiboCommerceConfig } from './../index'
+import { getCookieExpirationDate } from '@framework/lib/get-cookie-expiration-date'
```

Comment:
We should always do relative paths inside the providers folder, otherwise build will fail for other providers